api/envoy/config/filter/http/jwt_authn/v2alpha/config.proto

```diff
@@ -391,8 +391,7 @@ message FilterStateRule {
 
   // A map of string keys to requirements. The string key is the string value
   // in the FilterState with the name specified in the *name* field above.
-  map<string, JwtRequirement>
-  requires = 3;
+  map<string, JwtRequirement> requires = 3;
```

Comment:
I was told it's probably because my underlying Clang version is not in line with what Envoy uses so I will be re-installing Clang later. Regardless I actually can't even push past the format pre-checks from local to remote without doing this.